aspnetcore/fundamentals/error-handling.md

```diff
@@ -112,7 +112,7 @@ In addition to the exception handling logic in your app, the [server](servers/in
 
 Only the hosting layer can handle exceptions that take place during app startup. Exceptions that occur during app startup can impact server behavior. For example, if an exception happens before you call `KestrelServerOptions.UseHttps`, the hosting layer catches the exception, starts the server, and displays an error page on the non-SSL port. If an exception happens after that line executes, the error page is served over HTTPS instead.
```

Comment:
`KestrelServerOptions.UseHttps` is obsolete. I'm not sure this example is possible anymore. @halter73

Reply:
Now the call to UseHttps() happens before Kestrel attempts to bind to a port, so there can be no error page on the HTTPS port.